collect_app/src/androidTest/java/org/odk/collect/android/DeviceStateTest.java

```diff
@@ -0,0 +1,113 @@
+package org.odk.collect.android;
+
+import android.os.Build;
+import android.provider.Settings;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.preferences.AboutPreferencesActivity;
+
+import static org.junit.Assert.assertFalse;
+
+/**
+ * Tests to ensure the device is setup properly to run the tests. Before making changes to a test
+ * or researching a failure, make sure this set of tests pass.
+ */
+@RunWith(AndroidJUnit4.class)
+public class DeviceStateTest {
+
+    @Rule
+    public ActivityTestRule<AboutPreferencesActivity> mActivityRule =
+            new ActivityTestRule<>(
+                    AboutPreferencesActivity.class);
```

Comment:
I think we can just use MainMenuActivity? Is tehere any reason why we should use AboutPreferencesActivity?